libs/vue-plugin/src/schematics/application/files/src/router/index.ts.template

```diff
@@ -0,0 +1,14 @@
+import Vue from 'vue';
```

Comment:
I feel like this file should be generated inside of the `app` folder (app/routing/index.ts). What do you think?

Reply:
I agree.